### ISSUE TITLE
Variables environnement vs. variables settings Django

### ISSUE THREAD

**Issue by Skrattoune** [Skrattoune/OpenRepairPlatform#20]
Ca m'embête assez toutes ces constantes qui sont à définir sur le serveur et pas dans l'app

Je pense qu'il faudrait avoir une liste dans les fichiers docs de l'appli directement dans le repo

il y a d'ailleurs peut-être déjà

ex:
LOCATION = os.getenv("LOCATION", "False").lower() in ('true', '1', 'y')

**Comment by nfavrod** [comment]
Oui, il y a par exemple les variables de langue ici : https://github.com/Skrattoune/OpenRepairPlatform/blob/dev_branch/deployment/django/Dockerfile 

# French ENV for date
RUN sed -i -e 's/# fr_FR.UTF-8 UTF-8/fr_FR.UTF-8 UTF-8/' /etc/locale.gen && \
    locale-gen
ENV LC_ALL fr_FR
ENV LANG fr_FR
ENV LANGUAGE fr_FR
